static/public/scripts/script.js

```diff
@@ -19,19 +22,42 @@ function closeTip() {
 
 //sound eastereggs
 const likeButton = document.getElementById("bone");
-likeButton.addEventListener("click", sound);
-
 const dogImg = document.getElementById("dogImg");
-dogImg.addEventListener("click", sound2);
 
-function sound() {
-  var audio = new Audio("./files/nom.mp3");
+likeButton.addEventListener("click", () => {
+  sound("nom");
+});
+
+likeButton.addEventListener("mouseover", () => {
+  sound("sparkle");
+});
+
+dogImg.addEventListener("click", () => {
+  sound("bark");
+});
+
+//audio component
+function sound(file) {
+  let source = "./files/" + file + ".mp3";
+  let audio = new Audio(source);
   audio.volume = 0.05;
   audio.play();
 }
 
-function sound2() {
-  var audio = new Audio("./files/bark.mp3");
-  audio.volume = 0.1;
-  audio.play();
+const infoDog = document.getElementById("infodog");
+
+//jumping dog
+setInterval(hop, 4000);
+
+function hop() {
+  infoDog.style.marginBottom = "3em";
+  setTimeout(down, 100);
 }
+
+function down() {
+  infoDog.style.marginBottom = "0";
+}
+
+document.getElementById("del").addEventListener("click", () => {
+  alert("Wow. Did you really just delete a dog???!?!?1/1!/");
```

Comment:
Voor de volgende keer misschien geen alert maar een document.createElement of iets